scripts/gerrit_changed_files_to_skipfile.py

```diff
@@ -0,0 +1,54 @@
+# -------------------------------------------------------------------------
```

Comment:
```suggestion
#!/usr/bin/env python3
# -------------------------------------------------------------------------
```

Reply:
Yes I thought about it but the script is python2/python3 compatible now, with that change we limit it to python3 only do we want that?